src/CodeGen_LLVM.cpp

```diff
@@ -1340,6 +1340,9 @@ void CodeGen_LLVM::optimize_module() {
         }
     }
 
+    if (tm)
```

Comment:
Always enclose if stmts in brackets in Halide, even for single-line stmts

Reply:
Sorry. DId not know that. Corrected.